crates/binstalk/src/errors.rs

```diff
@@ -312,6 +312,22 @@ pub enum BinstallError {
     #[diagnostic(severity(error), code(binstall::SourceFilePath))]
     EmptySourceFilePath,
 
+    /// Invalid strategies configured.
+    ///
+    /// - Code: `binstall::strategies`
+    /// - Exit: 93
+    #[error("Invalid strategies configured: {0}")]
+    #[diagnostic(severity(error), code(binstall::strategies))]
+    InvalidStrategies(&'static &'static str),
+
+    /// Fallback to `cargo-install` is dissabled.
+    ///
+    /// - Code: `binstall::strategies`
```

Comment:
and the code is duplicated

Reply:
Change that to `no_fallback_to_cargo_install`.
P.S. If you think of a better code, feel free to request for change.